### ISSUE TITLE
Add "My Data" tab to user profile listing their contributions to the GBIF network

### ISSUE THREAD

**Issue by kbraak** [gbif/portal16#342]
Currently, GBIF.org user profiles have two tabs/pages of information: 
* "Account/Profile" tab - with basic account info
* "Downloads" tab - with list of downloads the user performed on GBIF.org

It would be great if a user's profile also had a "My Data" tab, that reflects their contributions to the GBIF network. Here are some example contributions and related statistics that could be shown:
* List of datasets that the user contributed to (determined for example by the presence of the user's ORCID or email address in dataset contact metadata)
* Statistics, showing the number of downloads using one or more records from these datasets
* Statistics, showing the pattern when these datasets were published over time  
* Statistics, showing the breakdown of types of these datasets (e.g. checklist, occurrence, etc)
* Statistics, showing the breakdown of the years in which the data was collected (e.g. based on temporal metadata) 

[DataOne](https://search.dataone.org/#data) user profiles include all these features listed above. More information about the DataOne screencast can be seen in [this screencast](http://vimeo.com/153160469).

Note DataOne users can sign in using their ORCID credentials. I hope GBIF will also support this for the reasons explained in issue https://github.com/gbif/portal16/issues/297 A demonstration of how DataOne sign in works with ORCID can be seen in [this screencast](http://vimeo.com/153160500). 

#### Current portal:
![screen shot 2017-04-10 at 15 42 35](https://cloud.githubusercontent.com/assets/5981100/24866471/7c18067c-1e0a-11e7-82fa-39a15cc1fd2c.png)
#### Demo portal:
![screen shot 2017-04-10 at 16 27 52](https://cloud.githubusercontent.com/assets/5981100/24866549/b91db9e0-1e0a-11e7-9491-97a4b2e14c20.png)

**Comment by MortenHofft** [comment]
I completely agree. Exactly what to put there isn't clear to me, but sounds like good ideas. Pending endorsements if you are a node manager perhaps. Helpdesk and content feedback related to your data. Some of the ideas require API changes as far as I can see, but something should be possible with the current state of the APIs as well.

**Comment by MortenHofft** [comment]
Just apart of a mockup - APIs is largely missing for any of these things to be doable. So if we decide to continue it needs to be a more concerted effort to get it done.
![profile dashboard-1](https://user-images.githubusercontent.com/6759267/33422549-5c195466-d5b6-11e7-95bc-44726f96d5d9.png)

**Comment by MortenHofft** [comment]
See also https://github.com/gbif/portal16/issues/297 for detailed discussions about what GBIF could/should do with user profiles
